url.py

```diff
@@ -0,0 +1,28 @@
+def remove_last_slash(url:str)->str:                    #Removes the last forward slash that is redundant to the url
+    if(url[-1]=='/'):
+        return(url[:-1])
+    else: 
+        return url 
+def get_protocol(url:str)->str:                         #Gets the protocol specified in the url
+    parsed_url = url.split('/')
+    protocol = parsed_url[0].replace(":",'')
+    return protocol
```

Comment:
split on `/` and then removing `:` seems to be unnecessary.

secondly, please handle the case when the input url doesn't have a protocol specified eg. `coeus-solutions.de`. In such case, use `http` as default.